nuxtjs3/components/TVChartContainer.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <div :id="containerId" class="TVChartContainer" />
+</template>
+
+<script>
+
+function getLanguageFromURL() {
+  // eslint-disable-next-line prefer-regex-literals
+  const regex = new RegExp('[\\?&]lang=([^&#]*)')
+  const results = regex.exec(window.location.search)
+  return results === null ? null : decodeURIComponent(results[1].replace(/\+/g, ' '))
+}
+
+export default {
+  name: 'TVChartContainer',
+  props: {
+    symbol: {
+      default: 'AAPL',
+      type: String
+    },
+    interval: {
+      default: 'D',
+      type: String
+    },
+    containerId: {
+      default: 'tv_chart_container',
+      type: String
+    },
+    datafeedUrl: {
+      default: 'https://demo_feed.tradingview.com',
+      type: String
+    },
+    libraryPath: {
+      default: '/charting_library/',
```

Comment:
I think we can do without the leading slash here.

Reply:
I'll update but to be fair I've lazily copied/pasted from the other nuxtjs example.

Reply:
This warning message is to help with cross-origin hosting.
```js
if (this._options.library_path && !this._options.library_path.endsWith('/')) {
	// tslint:disable-next-line:no-console
	console.warn('library_path option should contain a trailing forward slash');
}
```

So the ideal config is:
```js
library_path: "charting_library/",
```

It's very minor in this case but it will help if someone tries to host the chart on a sub-folder of a domain.